core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -748,7 +747,7 @@ private void applyConfiguration(CreateContainerCmd createCommand) {
         boolean shouldCheckFileMountingSupport = binds.size() > 0 && !TestcontainersConfiguration.getInstance().isDisableChecks();
         if (shouldCheckFileMountingSupport) {
             if (!DockerClientFactory.instance().isFileMountingSupported()) {
-                VisibleAssertions.warn(
+                System.err.println(
```

Comment:
Should we just make this use SLF4J? I know VisibleAssertions previously emitted to stdout, but it seems that we should probably take this opportunity to just consistently log to SLF4J...

Reply:
I guess we could for this one, yes :)